packages/strapi-design-system/src/Button/Button.js

```diff
@@ -1,5 +1,70 @@
+import React from 'react';
+import PropTypes from 'prop-types';
 import styled from 'styled-components';
+import { Text } from '../Text';
+import { Box } from '../Box';
+import { getDisabledStyle, getHoverStyle, getActiveStyle, getVariantStyle, getBoxPosition } from './utils';
+import { VARIANTS, BUTTON_SIZES } from './constants';
 
-export const Button = styled.button`
-  background: ${(props) => props.theme.colors.primary100};
+export const ButtonStyle = styled.button`
```

Comment:
```suggestion
export const ButtonWrapper = styled.button`
```

Since it's not a "style" per see, but a wrapper around the button 😊